tsa/tsacmd/command.go

```diff
@@ -141,6 +144,46 @@ func (cmd *TSACommand) Runner(args []string) (ifrit.Runner, error) {
 		httpClient:        httpClient,
 		sessionTeam:       sessionAuthTeam,
 	}
+	// Starts a goroutine that his purpose is to basically listen to the
+	// SIGHUP syscall to then reload the config.
+	// For now it only reload the TSACommand.AuthorizedKeys but any
+	// other configuration could be added
+	go func() {
+		for {
+
+			// Set up channel on which to send signal notifications.
+			// We must use a buffered channel or risk missing the signal
+			// if we're not ready to receive when the signal is sent.
+			c := make(chan os.Signal, 1)
+			signal.Notify(c, syscall.SIGHUP)
```

Comment:
I think it's best to pull this out of the loop - otherwise, the number of `chan`s that are being used to send signals to will grow without bound

Reply:
GC would collect them as they would not be used anymore but it's maybe safer or more explicite to put it out of the for loop.

I will push a commit for it